src/test/lib/template-result_test.ts

```diff
@@ -32,4 +33,15 @@ suite('TemplateResult', () => {
     const templateHTML = html`<div style="color: ${'red'}"></div>`.getHTML();
     assert.equal(templateHTML, `<div style$lit$="color: ${marker}"></div>`);
   });
+
+  test('invalid escape sequences always throw an error', () => {
+    function throws(template: TemplateResult) {
+      assert.throw(() => template.getHTML());
+    }
+
+    throws(html`<style> .foo:before { content: "\0025a0"; } </style>`);
+    throws(html`\0025a0${'bar'}\0025a0`);
+    throws(html`\0025a0${'bar'}foo`);
```

Comment:
I'm surprised this one throws an error due to my above comment.

Reply:
Explanation [here](https://github.com/Polymer/lit-html/pull/922#discussion_r287429628).